test/Microsoft.Azure.WebJobs.Host.UnitTests/Scale/FunctionsScaleMonitorServiceTests.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Microsoft.Azure.WebJobs.Host.UnitTests.Scale;
+using Microsoft.Extensions.Logging;
+
+using Moq;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Scale
+{
+    public class FunctionsScaleMonitorServiceTests
```

Comment:
Type name was changed, so tests should reflect that. Check for all the other tests as well.